src/library/librarycontrol.cpp

```diff
@@ -389,14 +388,16 @@ void LibraryControl::slotMoveFocus(double v) {
 }
 
 void LibraryControl::emitKeyEvent(QKeyEvent&& event) {
-    // Ensure a valid widget has the keyboard focus
+    // Ensure a valid library widget has the keyboard focus.
+    // QApplication::focusWidget() is not sufficient here because it
+    // would return any focussed widget like WOverview, WWaveform, QSpinBox
+    if (m_pLibraryWidget && !m_pLibraryWidget->hasFocus() &&
+            m_pSidebarWidget && !m_pSidebarWidget->hasFocus()) {
+        setLibraryFocus();
+    }
     auto focusWidget = QApplication::focusWidget();
     if (!focusWidget) {
```

Comment:
I wonder if this check is still necessary after we've tested for library & sidebar widget above?

Reply:
The invocation context might change in the future.. Use a separate check for each pointer, just to be sure:

```
VERIFY_OR_DEBUG_ASSERT(...) {
    return;
}
```